int16 len
float64[] traj
---
